### ISSUE TITLE
TextFieldCornerRadius doesn't work with combobox with HasFilledTextField=True

### ISSUE THREAD

**Issue by frostiger** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2503]
The `TextFieldAssist.TextFieldCornerRadius` doesn't work with a combobox wich has `TextFieldAssist.HasFilledTextField` enabled (since version 4.1.0; in version 4.0.0 everything works fine). The filled textfield overlaps in the corners or the corner radius is not applied to the textfield.

This is the definition of the combobox:
```
<ComboBox
Width="200"
Padding="15"
materialDesign:TextFieldAssist.TextFieldCornerRadius="20"
materialDesign:TextFieldAssist.HasFilledTextField="True"
Background="GreenYellow"
BorderThickness="1" />
```
The values for `Width`, `Padding`, `Background` and `BorderThickness` are not relevant to the problem, but only serve to improve the presentation.

| v4.0.0 | v4.1.0 and above |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61822425/143765027-dd479035-e7ac-4158-9b86-8523aad7921b.png)  | ![image](https://user-images.githubusercontent.com/61822425/143765046-b64ab0eb-5a7b-4540-a934-1eb066224bff.png)  |

**Comment by MichelMichels** [comment]
Could you show this in a git repository? I can't replicate this, but maybe it's already fixed.

**Comment by frostiger** [comment]
Here's a repository showing the issue:
[https://github.com/frostiger/MaterialDesignComboboxIssue]

**Comment by Keboo** [comment]
Thank you for the repro project.
Confirmed fixed on the latest nightly release 4.4.0-ci38-ci38.

**Comment by frostiger** [comment]
> Thank you for the repro project. Confirmed fixed on the latest nightly release 4.4.0-ci38-ci38.

Thank you!
